lib/live_sup_web/helpers.ex

```diff
@@ -125,4 +125,17 @@ defmodule LiveSupWeb.Helpers do
     </div>
     """
   end
+
+  def page_head(assigns) do
+    import Palette.Components.Head, only: [head: 1]
```

Comment:
@mustela i have a question about this, i tried to use it as an alias but that didn't work, the only way was using the import but didn't find documentation about why the "alias" didn't work.